azurelinuxagent/ga/controllermetrics.py

```diff
@@ -169,10 +169,16 @@ def get_tracked_metrics(self, **_):
         """
         raise NotImplementedError()
 
+    def get_unit_properties(self):
```

Comment:
Yeah they are only being used to log the properties when we initialize cgroups. We log the properties before setting any quotas. Then after setting cpu quota, we log only the cpuquota property value. 

It was logging we were doing before so I kept it, but since we log the properties after setting quotas it might not be necessary.

Reply:
maybe remove them altogether? the log message doesn't have any context about what those are

or add more context to the message, if you think this could be useful

Reply:
I'll open a PR to improve the message. I think it may be useful to include, so that we can see from the logs if there were any quotas previously set before the current agent attempts to set quotas